sdks/java/core/src/main/java/org/apache/beam/sdk/io/BigQueryIO.java

```diff
@@ -2393,6 +2370,13 @@ static String toJsonString(Object item) {
     }
   }
 
+  /**
+   * Returns a randomUUID string without {@code '-'}.
+   */
+  private static String randomUUIDString() {
```

Comment:
how about "randomJobId"

And then in the comments explain why you remove the '-'